internal/helpers.go

```diff
@@ -87,7 +87,7 @@ type cobraTestRunner struct {
 }
 
 func consumeLines(ctx context.Context, wg *sync.WaitGroup, r io.Reader) <-chan string {
-	ch := make(chan string, 1000)
```

Comment:
Did this panic or hang?

Reply:
It panicked with `line buffer is full` error